elyra/pipeline/tests/resources/sample_pipelines/pipeline_3_node_sample.json

```diff
@@ -14,7 +14,14 @@
           "op": "execute-notebook-node",
           "app_data": {
             "filename": "demo-pipelines/generate-contributions.ipynb",
-            "runtime_image": "elyra/tensorflow:1.15.2-py3"
+            "runtime_image": "elyra/tensorflow:1.15.2-py3",
+            "ui_data": {
+              "label": "{{label}}",
+              "image": "data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2216%22%20viewBox%3D%220%200%2022%2022%22%3E%0A%20%20%3Cg%20class%3D%22jp-icon-warn0%20jp-icon-selectable%22%20fill%3D%22%23EF6C00%22%3E%0A%20%20%20%20%3Cpath%20d%3D%22M18.7%203.3v15.4H3.3V3.3h15.4m1.5-1.5H1.8v18.3h18.3l.1-18.3z%22%2F%3E%0A%20%20%20%20%3Cpath%20d%3D%22M16.5%2016.5l-5.4-4.3-5.6%204.3v-11h11z%22%2F%3E%0A%20%20%3C%2Fg%3E%0A%3C%2Fsvg%3E%0A",
```

Comment:
do we need to add an image field to the json?

Reply:
I can remove it since its just test material